packages/contracts-bedrock/contracts/test/SafeCall.t.sol

```diff
@@ -39,4 +39,117 @@ contract SafeCall_call_Test is CommonTest {
         assertEq(to.balance, value, "to balance received");
         assertEq(from.balance, 0, "from balance not drained");
     }
+
+    function testFuzz_callWithMinGas_hasEnough_succeeds(
+        address from,
+        address to,
+        uint64 minGas,
+        uint64 value,
+        bytes memory data
+    ) external {
+        vm.assume(from.balance == 0);
+        vm.assume(to.balance == 0);
+        // no precompiles
+        vm.assume(uint160(to) > 10);
```

Comment:
There is a helper in forge-std for no precompiles
https://github.com/foundry-rs/forge-std/blob/c2236853aadb8e2d9909bbecdc490099519b70a4/src/StdCheats.sol#L200

Reply:
I wonder how many of these `vm.assume()` calls are boiler plate that can be moved into a single helper function (something for another PR).